src/common/Utilities.ts

```diff
@@ -41,8 +43,8 @@ export default class Utilities {
                 finalPath += "?";
             }
             for (const queryParamKey in queryParameters) {
-                if (queryParameters.hasOwnProperty(queryParamKey)) {
-                    let queryParamData = queryParamKey + "=" + queryParameters[queryParamKey];
+                if (queryParameters.hasOwnProperty(queryParamKey)) { // eslint-disable-line no-prototype-builtins
+                    let queryParamData = queryParamKey + "=" + queryParameters[queryParamKey]; // eslint-disable-line security/detect-object-injection
```

Comment:
Could we change this to to use a template string, a la `${queryParamKey}=${queryParameters[queryParamKey]}`?

Reply:
Actually nevermind.